mvm/connector.yaml

```
@@ -32,7 +32,7 @@ services:
       WEB_HTTP_PORT: ${CONN_WEB_HTTP_PORT:-18181}
       WEB_HTTP_PATH: ${CONN_WEB_HTTP_PATH:-/api}
       EDC_CONTROL_ENDPOINT: http://connector-dataplane:${CONN_WEB_HTTP_CONTROL_PORT}${CONN_WEB_HTTP_CONTROL_PATH}
-      EDC_DSP_CALLBACK_ADDRESS: http://${SED_PUBLIC_HOSTNAME}:${CONN_CP_PUBLIC_DSP_PORT}${CONN_WEB_HTTP_PROTOCOL_PATH}
+      EDC_DSP_CALLBACK_ADDRESS: https://${CONN_DATA_PLANE_HOSTNAME}:${CONN_DATA_PLANE_PUBLIC_PORT}${CONN_DATA_PLANE_PUBLIC_PATH}
```

Two comments here:
- as you mentioned in your original post, it would be better to parametrize the protocol (`http`/`https`), so everyone gains in flexibility.
- I think this should be the following, considering that it is called _DSP callback_. @juanrasantana or @vgonzalez7 should have a look to confirm
```suggestion
      EDC_DSP_CALLBACK_ADDRESS: https://${CONN_CONTROL_PLANE_HOSTNAME}:${CONN_CONTROL_PLANE_PORT}${CONN_CONTROL_PLANE_PATH}
```

---

Yes, the previous variable were also pointing to the control-plane. (easy to get mixed up with some of this names.)

---

What Maxime said is correct, the DSP is used by the control plane.